spec/controllers/admin/users_controller_spec.rb

```diff
@@ -3,10 +3,35 @@
 describe Admin::UsersController do
 
   describe 'GET index' do
-    subject { get :index }
+    subject(:get_index) { get :index }
 
     authenticated_as(:admin) do
       it { should be_success }
+
+      describe "populating @users" do
+        subject(:users) { get_index; assigns(:users) }
+
+        it { should include(FactoryGirl.create(:user, :member)) }
+        it { should_not include(FactoryGirl.create(:user, :admin)) }
+      end
+    end
+
+    it_behaves_like "action requiring authentication"
+    it_behaves_like "action authorizes roles", [:admin]
+  end
+
+  describe 'GET index_admins' do
+    subject(:get_index) { get :index_admins }
+
+    authenticated_as(:admin) do
+      it { should be_success }
+
+      describe "populating @users" do
+        subject(:users) { get_index; assigns(:users) }
+
+        it { should include(FactoryGirl.create(:user, :admin)) }
+        it { should_not include(FactoryGirl.create(:user, :member)) }
```

Comment:
The only reason .should was removed was so there was no monkey patching, and the only reason is_expected was added was to be consistent with .expect

I don't think should and expect are inconsistent at all, they are used for different purposes.